templates/scripts/add-vpn-user.sh

```diff
@@ -6,25 +6,26 @@ CLUSTER=$(kubectl config current-context | cut -d"/" -f2)
 
 # get pod id for execution
 POD=$(kubectl -n vpn get pods | grep wireguard | cut -d' ' -f1)
+EXTERNAL_DNS=$(kubectl -nvpn get svc wireguard -o jsonpath='{.metadata.annotations.external-dns\.alpha\.kubernetes\.io/hostname}')
 
 if [ -z "$POD" ]; then
   echo "Warning: No VPN service running yet"
   exit 1
 fi
-EXEC="kubectl -n vpn exec -it $POD --"
+EXEC="kubectl -n vpn exec -it $POD -- /bin/bash -c"
 
 # get name
 echo -n "Enter your name: "
 read name
 
 # collect keys
-server_public_key=$($EXEC cat /etc/wireguard/privatekey | wg pubkey)
-client_private_key=$($EXEC wg genkey)
-client_public_key=$($EXEC echo -n $client_private_key | wg pubkey)
+server_public_key=$($EXEC "cat /etc/wireguard/privatekey | wg pubkey")
+client_private_key=$($EXEC "wg genkey")
+client_public_key=$($EXEC "echo -n $client_private_key | wg pubkey")
 
 # get next available IP
-existing_ips=$($EXEC cat /etc/wireguard/wg0.conf | grep AllowedIPs| cut -d" " -f3 | cut -d"/" -f1 | sort)
-last_ip=$(echo "$existing_ips" | tail -1)
+existing_ips=$($EXEC "cat /etc/wireguard/wg0.conf | grep AllowedIPs| cut -d\" \" -f3 | cut -d\"/\" -f1 | sort")
+last_ip=$(echo "$existing_ips" | tr -cd "[:alnum:]." | tail -1)
```

Comment:
somehow when it doesnt work for me unless I sanitize the string `tr -cd "[:alnum:]."` 
I've tried to pipe it into a file and look at it but it looks fine